File: quadratic equation.py
# PREPARING QUADRATIC EQUATION AND FINDING ITS ROOTS
# THE VALUSE OF A,B,C ARE THE COEFFICIENTS OF X^2,X,CONSTANT
# RESPECTIVELY.THE USER ALIGNS THE VALUES OF THEM.
print("ENTER THE COEFFICIENT OF X^2")
a = int(input())
print("ENTER THE COEFFICIENT OF X")
b = int(input())
print("ENTER THE CONSTANT")
c = int(input())

# THE QUADRIC EQUYATION WILL BE FORMED
print("THE QUADRATIC EQUATION IS " + str(a) + "X^2+" + str(b) + "X+" + str(c))

# THYE SYSTEM WILL DISPLAYS THE ROOTS OF THE EQUATION
root1 = (-b + (b ** 2 - 4 * a * c) ** (float(1) / 2)) / 2 * a
print("THE ROOT1 " + str(root1))
root2 = (-b - (b ** 2 - 4 * a * c) ** (float(1) / 2)) / 2 * a
print("THE ROOT2 " + str(root2))

File: C/quadratic equation.c
#include <stdio.h>
#include <math.h>

int main() {
    int a, b, c,d;
    float root1, root2;

    printf("ENTER THE COEFFICIENT OF X^2: ");
    scanf("%d", &a);
    printf("ENTER THE COEFFICIENT OF X: ");
    scanf("%d", &b);
    printf("ENTER THE CONSTANT: ");
    scanf("%d", &c);

    // THE QUADRATIC EQUATION WILL BE FORMED
    printf("THE QUADRATIC EQUATION IS %dX^2+%dX+%d\n", a, b, c);
    d=b*b-4*a*c;

    // THYE SYSTEM WILL DISPLAYS THE ROOTS OF THE EQUATION
    root1 = (-b + sqrt(d)) / 2 * a;
    printf("THE ROOT1 %f\n", root1);
    root2 = (-b - sqrt(d)) / 2 * a;
    printf("THE ROOT2 %f\n", root2);
return 0;
}
